nixos/modules/services/networking/prosody.nix

```diff
@@ -422,6 +422,13 @@ in
         description = "List of administrators of the current host";
       };
 
+      authentication = mkOption {
+        type = types.str;
+        default = "internal_hashed";
+        example = "internal_plain";
+        description = "Authentication mechanism used for logins";
```

Comment:
Needs a `.` at the end (by convention :))

Reply:
Fixed :)